lib/language.js

```diff
@@ -422,6 +473,7 @@ function init() {
       ,fr: 'Percentiles'
       ,pt: 'Percentis'
       ,ro: 'Grafic percentile'
+      ,бг: "Процентна графика"
```

Comment:
Oh sorry I must have done it by mistake . 
I hope there not many like that . I will also check it . 
Hristina 

> On 30.07.2015 г., at 16:42, Jason Calabrese <notifications@github.com> wrote:
> 
> In lib/language.js:
> 
> > @@ -422,6 +473,7 @@ function init() {
> >        ,fr: 'Percentiles'
> >        ,pt: 'Percentis'
> >        ,ro: 'Grafic percentile'
> > +      ,бг: "Процентна графика"
>  Issue found: Mixed double and single quotes.
> 
> —
> Reply to this email directly or view it on GitHub.
>